scrapers/getStates.js

```diff
@@ -11,7 +11,6 @@ const parseStateCell = (cell) => {
 		|| '';
 	state = state.trim();
 	if (state.length === 0) {
-		// parse with hyperlink
 		state = cell.children[0].next.children[0].data || '';
 	}
 	return state.trim() || '';
```

Comment:
couldnt we use falsy values to cut this down to
`return state.trim() || cell.children[0].next.children[0].data.trim() || '';`

Reply:
also line 42-50 could be reduced down to
`return tableRows.map((row) => {
		const cells = row.children.filter((cell) => cell.name === 'td');
		const stateData = { state: parseStateCell(cells[stateColIndex]) };
		Object.keys(dataColIndexes).forEach((property) => {
			stateData[property] = parseNumberCell(cells[dataColIndexes[property]]);
		});
		return stateData;
	});`